components/calendar/src/cal/hijri.rs

```diff
@@ -247,8 +251,10 @@ impl Rules for AstronomicalSimulation {
 /// [KACST](https://kacst.gov.sa/). Outside this range, this implementation falls back to
 /// [`TabularAlgorithm`] with [`TabularAlgorithmLeapYears::TypeII`] and [`TabularAlgorithmEpoch::Friday`].
 ///
-/// Future versions of this crate may extend the range that uses month length data from the
-/// calendar authority.
+/// The precise behavior of this calendar may change in the future if:
+/// - New ground truth is established by published government sources
+/// - We decide to tweak the simplified calculation
```

Comment:
there's no "simplified calculation" here, the fallback is tabular. 
```suggestion
/// - We decide to use a different algorithm outside the KACST range
```